### PR TITLE
docs: update readme after new root signing

### PR DESCRIPTION
**Pull request by asraa**
Signed-off-by: Asra Ali <asraa@google.com>

See title. Adds @joshuagl's new key, and updates targets. 
Adds all top-level key IDs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,18 +33,20 @@ The current published repository metadata lives in the [repository](/repository/
...
 The current root is published on a GCS bucket located at `https://storage.googleapis.com/sigstore-tuf-root`.
 
+The pre-production root is published ona GCS bucket located at `https://storage.googleapis.com/sigstore-preprod-tuf-root`.
+
+
+## Sigstore Root Keyholders 
+
+| Keyholder        |  TUF Key ID |  Yubikey Material| Term | 
+| ----- |--------- |  --- | ---- |
+| Joshua Locke       |  `75e867ab10e121fdef32094af634707f43ddd79c6bab8ad6c5ab9f03f4ea8c90` | [18158855](https://github.com/sigstore/root-signing/ceremony/2022-07-12/keys/18158855)  | July 2022 -  |
```
Comment (joshuagl) [inline README.md]:
```suggestion
| Joshua Lock       |  `75e867ab10e121fdef32094af634707f43ddd79c6bab8ad6c5ab9f03f4ea8c90` | [18158855](https://github.com/sigstore/root-signing/ceremony/2022-07-12/keys/18158855)  | July 2022 -  | 
```
no e :-)

File: README.md [hunk 2]
```diff
@@ -4,17 +4,27 @@ This directory contains the programs needed to generate and verify Sigstore root
...
+
+* [targets.json](repository/repository/targets.json): This is the list of trusted `targets.json` endorsed by the offline keyholders. It includes:
+
+| Target        |  Description | 
+| ----- |--------- |  
+| [fulcio_v1.crt.pem](repository/repository/targets/fulcio_v1.crt.pem)   |  This is the [Fulcio](https://github.com/sigstore/fulcio) root certificate used to issue short-lived code signing certs. It is hosted at `https://fulcio.sigstore.dev`. You can `curl` the running root CA to ensure it matches the TUF root using `curl -v https://fulcio.sigstore.dev/api/v1/rootCert` | 
+| [fulcio_intermediate_v1.crt.pem](repository/repository/targets/fulcio__intermediate_v1.crt.pem)   |  This is the [Fulcio](https://github.com/sigstore/fulcio) intermediate certificate used to issue short-lived code signing certs. It is hosted at `https://fulcio.sigstore.dev`. You can `curl` the running CA chain to ensure it matches the TUF root using `curl -v https://fulcio.sigstore.dev/api/v1/rootCert` |
```
Comment (joshuagl) [inline README.md]: The URL in the curl example here is the same as the entry above

File: README.md [hunk 3]
```diff
@@ -4,17 +4,27 @@ This directory contains the programs needed to generate and verify Sigstore root
...
+
+| Target        |  Description | 
+| ----- |--------- |  
+| [fulcio_v1.crt.pem](repository/repository/targets/fulcio_v1.crt.pem)   |  This is the [Fulcio](https://github.com/sigstore/fulcio) root certificate used to issue short-lived code signing certs. It is hosted at `https://fulcio.sigstore.dev`. You can `curl` the running root CA to ensure it matches the TUF root using `curl -v https://fulcio.sigstore.dev/api/v1/rootCert` | 
+| [fulcio_intermediate_v1.crt.pem](repository/repository/targets/fulcio__intermediate_v1.crt.pem)   |  This is the [Fulcio](https://github.com/sigstore/fulcio) intermediate certificate used to issue short-lived code signing certs. It is hosted at `https://fulcio.sigstore.dev`. You can `curl` the running CA chain to ensure it matches the TUF root using `curl -v https://fulcio.sigstore.dev/api/v1/rootCert` | 
+| [fulcio.crt.pem](repository/repository/targets/fulcio.crt.pem)        |  This is the Fulcio root certificate used with an older instance of Fulcio. We maintain this target to verify old certificates but no longer used to sign newly issued certificates. |
```
Comment (joshuagl) [inline README.md]:
```suggestion
| [fulcio.crt.pem](repository/repository/targets/fulcio.crt.pem)        |  This is the Fulcio root certificate used with an older instance of Fulcio. We maintain this target to verify old certificates but it is no longer used to sign newly issued certificates. | 
```

File: README.md [hunk 4]
```diff
@@ -23,18 +33,20 @@ The current published repository metadata lives in the [repository](/repository/
 
 The current root is published on a GCS bucket located at `https://storage.googleapis.com/sigstore-tuf-root`.
 
+The pre-production root is published ona GCS bucket located at `https://storage.googleapis.com/sigstore-preprod-tuf-root`.
```
Comment (joshuagl) [inline README.md]:
```suggestion
The pre-production root is published on a GCS bucket located at `https://storage.googleapis.com/sigstore-preprod-tuf-root`.
```

File: README.md (hunk 1 above)
Comment (asraa) [reply README.md]: Ah thank you! I don't know why I did that.

File: README.md (hunk 2 above)
Comment (asraa) [reply README.md]: Yeah, this returns both the root and the intermediate. I will say to match with the first/second PEM cert.

File: README.md [hunk 5]
```diff
@@ -23,18 +33,20 @@ The current published repository metadata lives in the [repository](/repository/
...
+| Bob Callaway        |  `f505595165a177a41750a8e864ed1719b1edfccd5a426fd2c0ffda33ce7ff209` | [15938791](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/15938791)  | June 2021 -  | 
+| Dan Lorenc        |  `2f64fb5eac0cf94dd39bb45308b98920055e9a0d8e012a7220787834c60aef97` | [13078778](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/13078778)  | June 2021 -  | 
+| Marina Moore        |  `eaf22372f417dd618a46f6c627dbc276e9fd30a004fc94f9be946e73f8bd090b` | [14470876](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/14470876)  | June 2021 -  | 
+| Santiago Torres-Arias        |  `f40f32044071a9365505da3d1e3be6561f6f22d0e60cf51df783999f6c3429cb` | [15938765](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/15938765) | June 2021 -  | 
+| Luke Hinds        |  `bdde902f5ec668179ff5ca0dabf7657109287d690bf97e230c21d65f99155c62` | [14454335](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/14454335)  | June 2021 - July 2022 |
```
Comment (Hayden-IO) [inline README.md:54]: I think we should keep the separation of current and previous.

**Review by Hayden-IO — DISMISSED** [review]

File: README.md [hunk 6]
```diff
@@ -4,17 +4,27 @@ This directory contains the programs needed to generate and verify Sigstore root
...
+| [rekor.pub](repository/repository/targets/rekor.pub)        |  This is the [Rekor](https://github.com/sigstore/rekor) public key used to sign entries and the tree head of the transparency log. You can retrieve the public key to ensure it matches with `curl -H 'Content-Type: application/x-pem-file' https://rekor.sigstore.dev/api/v1/log/publicKey`. | 
+| [rekor.0.pub](repository/repository/targets/rekor.0.pub)        |  This is a dupe of `rekor.pub` and will be removed in the next root-signing event. | 
+| [ctfe.pub](repository/repository/targets/ctfe.pub)        |  Certificate Transparency log key that is used for certificates issued by Fulcio and used to verify signed certificate timestamps (SCTs) for inclusion into the log. | 
+| [artifact.pub](repository/repository/targets/artifact.pub) | Key that signs Sigstore project (Cosign, Rekor, Fulcio) releases. |
+
 * [snapshot.json]((repository/repository/snapshot.json)): This snapshot the valid metadata files. It has a lifetime of 2 weeks and is resigned by a [GitHub workflow](https://github.com/sigstore/root-signing/blob/main/.github/workflows/snapshot-timestamp.yml).
```
Comment (joshuagl) [inline README.md]:
```suggestion
* [snapshot.json]((repository/repository/snapshot.json)): The snapshot ensures consistency of the metadata files. It has a lifetime of 2 weeks and is re-signed by a [GitHub workflow](https://github.com/sigstore/root-signing/blob/main/.github/workflows/snapshot-timestamp.yml).
```

File: README.md [hunk 7]
```diff
@@ -4,17 +4,27 @@ This directory contains the programs needed to generate and verify Sigstore root
...
 * [timestamp.json]((repository/repository/timestamp.json)): The timestamp refreshes the most accurate snapshot file. It has a lifetime of 2 weeks and is resigned by a [GitHub workflow](https://github.com/sigstore/root-signing/blob/main/.github/workflows/snapshot-timestamp.yml).
```
Comment (joshuagl) [inline README.md]:
```suggestion
* [timestamp.json]((repository/repository/timestamp.json)): The timestamp indicates the freshness of the metadata files. It has a lifetime of 2 weeks and is re-signed by a [GitHub workflow](https://github.com/sigstore/root-signing/blob/main/.github/workflows/snapshot-timestamp.yml).
```

**Review by joshuagl — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (joshuagl) [reply README.md]: I should have tried and learned that before commenting, thanks!

**Review by joshuagl — APPROVED** [review]

**Pull request merged**